test/test_restful.py

```diff
@@ -275,7 +282,14 @@ def initthread():
                 sys.stdout.write('\n\033[94m' + line + '\033[0m')
         t = threading.Thread(target=initthread)
         t.start()
-        time.sleep(10)
+        for retry in range(10):
+            r = subprocess.run(["ss", "-l", "-t", "-n", "-p",  "( sport = :9002 )"], stdout=subprocess.PIPE)
+            if b":9002" in r.stdout:
+                break
+            elif retry == 10:
+                raise Exception(f"{agent} keylime_agent failed to launch")
+            time.sleep(1)
+        #time.sleep(10)
```

Comment:
You can just remove this commented out code. It's safe in the git history.

Reply:
Done!